Source/Blazorise/Components/Table/TableRow.razor

```diff
@@ -2,6 +2,8 @@
 @inherits BaseDraggableComponent
 <tr id="@ElementId" class="@ClassNames" style="@StyleNames" @onclick="@OnClickHandler"
     draggable="@DraggableString"
+    @onmouseover="@OnMouseOverHandler"
```

Comment:
Do we need `:preventDefault` for these events?

Reply:
It doest seem like it